### PR TITLE
Merge docs repo into this repo: add /docs

### PR DESCRIPTION
**Pull request by ndelangen**
Issue:

## What I did
- I merged the 3.0.0#source branch for docs into a /docs folder in this repo.
- More updates to reflect 3.0.0
- Linting for markdown on this entire repo

## How to test
It's already deployed to http://storybook.js.org

### CODE REVIEW

**Comment by ndelangen** [comment]
@mnmtanish @shilman @mthuret  Would you like to review this?

File: addons/actions/package.json [hunk 1]
```diff
@@ -21,22 +21,19 @@
   },
   "homepage": "https://github.com/storybooks/storybook/tree/master/addons/actions",
   "devDependencies": {
-    "@storybook/addons": "^3.0.0-rc.0",
```
Comment (shilman) [inline addons/actions/package.json]: Unrelated to docs, but we'll let it slide 😉

**Review by shilman — APPROVED** [review]
LGTM! Fingers crossed!

**Comment by ndelangen** [comment]
It would appear like a deploy to gh-pages wasn't succesful, can you look into it @shilman

File: addons/actions/package.json (hunk 1 above)
Comment (Hypnosphi) [reply addons/actions/package.json]: @ndelangen Is there a chance that you can remember why was it needed to move `@storybook/addons` frop peerDeps to direct deps?

Comment (ndelangen) [reply addons/actions/package.json]: Absolutely 0 chance of me remembering, sorry.

Though I think it makes sense, and without lock-files would work great.
Having these lockfiles really messes up the singleton module method, which lib/addons depends on to function.

Comment (Hypnosphi) [reply addons/actions/package.json]: > without lock-files would work great

But it's probably the only reason users have to use the same **patch** version for all the `@storybook` dependencies

**Pull request merged**